l10n_ch_import_camt054/models/custom_complete_stmt.py

```diff
@@ -0,0 +1,25 @@
+from odoo import api, models
+
+
+class AccountStatementImportCustomCamt053(models.TransientModel):
+    _inherit = 'account.bank.statement.import'
+
+    @api.model
+    def _complete_stmts_vals(self, stmts_vals, journal, account_number):
+        """
+        Use completion rules in journal to find missing partners.
+        """
+        stmts_vals = super(AccountStatementImportCustomCamt053, self)._complete_stmts_vals(
+            stmts_vals, journal, account_number)
+
+        list_transactions = stmts_vals[0]['transactions']
+
+        for transaction in list_transactions:
+            if transaction['sub_fmly_cd'] == 'RRTN':
```

Comment:
use `transaction.get('sub_fmly_cd')` to avoid errors in case sub_fmly_cd is not in the transaction values.